tests/suites/0_stateless/20+_others/20_0006_system_tables_with_dropped.sql

```diff
@@ -0,0 +1,13 @@
+DROP DATABASE IF EXISTS db_20_0005;
```

Comment:
Great feature!
I think all tests related to the`UNDROP` is the DDL test, so it would be better to move them to `05_ddl` test folder.

Reply:
These cases used to be there (0_stateless/05_ddl/05_0000_ddl_create_tables..).

But unfortunately, github action `test_compat` will also run `05_0000_ddl_create_tables` by using  previous version of databend-query (hope I get it right), which do not have the column `dropped_on` yet.

pls also see:
https://github.com/datafuselabs/databend/blob/ec1db2cb6d295f21e461fe5634e7744bab6e1723/tests/suites/0_stateless/05_ddl/05_0000_ddl_create_tables.sql#L8-L14

Reply:
or shall we put these cases into a new dir, say `12_time_travel`